frontend/src/metabase/query_builder/selectors.js

```diff
@@ -276,6 +276,7 @@ export const getTables = createSelector(
 
     return [];
   },
+  { devModeChecks: { inputStabilityCheck: "never" } },
```

Comment:
:x: This warning is correct and can be fixed by returning the same empty array in line 277.